pkg/adapter/kubernetesevents/cloudevent.go

```diff
@@ -75,10 +81,14 @@ func createSelfLink(o corev1.ObjectReference) string {
 //		Type:Normal,
 //		EventTime:0001-01-01 00:00:00 +0000 UTC,
 //	}
-func cloudEventOverrides(m *corev1.Event) cloudevents.V01EventContext {
-	return cloudevents.V01EventContext{
-		EventID:   string(m.ObjectMeta.UID),
-		Source:    createSelfLink(m.InvolvedObject),
-		EventTime: m.ObjectMeta.CreationTimestamp.Time,
+func cloudEventFrom(m *corev1.Event) cloudevents.Event {
+	return cloudevents.Event{
+		Context: cloudevents.EventContextV01{
```

Comment:
It would be nice to have a factory here which could default the EventID (and possibly time, though you'd want to override that here).

Do you want this to be a CE V0.1 context, or use the V0.2 names here?

Reply:
The new lib does not care what you give it, only what the transport is set to. So this results in the same thing. Maybe magic, maybe neat?

Reply:
I know it results in the same thing. 😉

 I'm wondering if you want to use the newer names so that newer readers aren't too confused. 😁

Reply:
got it, yes! Done.